specification/maps/data-plane/Search/preview/2022-09-01-preview/search.json

```diff
@@ -0,0 +1,966 @@
+      "type": "string",
+      "enum": [
+        "High",
+        "Medium",
+        "Low"
+      ],
+      "x-ms-enum": {
+        "name": "ConfidenceEnum",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "High"
+          },
+          {
+            "value": "Medium"
+          },
+          {
+            "value": "Low"
+          }
+        ]
+      }
+    },
+    "NextLink": {
+      "type": "string",
+      "description": "The is the link to the next page of the features returned. If it's the last page, no this field."
+    },
+    "FeatureType": {
+      "type": "string",
+      "enum": [
+        "FeatureCollection"
+      ],
+      "x-ms-enum": {
+        "name": "FeatureTypeEnum",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "FeatureCollection"
+          }
+        ]
+      }
+    },
+    "Boundary": {
+      "description": "`GeoJSON GeocodingFeature` object that describe the boundaries of a geographical area. Geometry of the feature is described with `GeoJSON GeometryCollection`.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/definitions/GeoJsonFeature"
+        },
+        {
+          "$ref": "#/definitions/BoundaryProperties"
+        }
+      ]
+    },
+    "BoundaryProperties": {
+      "description": "Properties of a Boundary object.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name associated with the geographical area.",
+          "type": "string"
+        },
+        "copyright": {
+          "description": "The copyright string.",
+          "type": "string"
+        },
+        "copyrightURL": {
+          "description": "A URL that lists many of the data providers for Azure Maps and their related copyright information.",
+          "type": "string"
```

Comment:
FYI I tried adding "format: uri" and "format: url", but I get an error when running validation locally that the "uri" is not known format.